marimo/_ast/names.py

```diff
@@ -0,0 +1,13 @@
+import hashlib
+
+DEFAULT_CELL_NAME = "__"
+
+
+def is_default_cell_name(name: str) -> bool:
+    return name.startswith("__") and (name == "__" or len(name) == 6)
+
+
+def default_cell_name_from_code(code: str) -> str:
+    return (
+        f"{DEFAULT_CELL_NAME}{hashlib.sha256(code.encode()).hexdigest()[:4]}"
```

Comment:
Should this be 6 characters to decrease the chance of a spurious collision? Birthday paradox is sneaky.

Reply:
I can increase to six, https://marimo.app/l/d4zs5k

<img width="779" alt="image" src="https://github.com/user-attachments/assets/f5f25cb9-b4a4-4094-9f24-dc97bab9b11b" />